### PR TITLE
Issue 13 Refactored Account capabilities to match the spec

### PR DESCRIPTION
**Pull request by ddolcimascolo**

### CODE REVIEW

File: lib/models/Account.js [hunk 1]
```diff
@@ -29,7 +35,37 @@ export default class Account extends Model {
...
+    this.calendars = Utils._nullOrNewInstance(opts.calendars, Capabilities);
+  }
+
+  /**
+   * Returns whether this `Account` supports mail or not.
+   *
+   * @returns {Boolean} _true_ if and only if this `Account` has mail enabled, _false_ otherwise.
+   */
+  hasMail() {
+    return this.mail !== null;
```
Comment (thomas-hilaire) [inline lib/models/Account.js:50]: don't you want `return !!this.mail;` ?

Comment (ddolcimascolo) [inline lib/models/Account.js:50]: I thought about it, but the spec mandates `null` so I'd rather check that... Do you think it's a problem?

Comment (rpignolet) [inline lib/models/Account.js:50]: Personally I prefer `return this.mail !== null;`

Comment (thomas-hilaire) [inline lib/models/Account.js:50]: I'm ok with both :+1:

**Pull request merged**